Makefile

```diff
@@ -211,7 +213,7 @@ test-cpp: build-test-cpp
 
 test-java: build-test-java
 	echo "Running tests against $(LIB_PROFILER)"
-	$(JAVA) "-Djava.library.path=$(TEST_LIB_DIR)" $(TEST_FLAGS) -ea -cp "build/test.jar:build/jar/*:build/lib/*" one.profiler.test.Runner $(TESTS)
+	$(JAVA) "-Djava.library.path=$(TEST_LIB_DIR)" $(TEST_FLAGS) -ea -cp "build/test.jar:build/jar/*:build/lib/*" one.profiler.test.Runner $(TEST_DIRS) $(TEST_FILTER)
```

Comment:
There are still two different variables. My idea was to have one, so a user can specify `TESTS=cpu` or `TESTS=ClassName` or `TESTS=ClassName.methodName`. No filters - just a list of tests to run. If it helps, make test specifier case-sensitive, so it's easy to disambiguate between directory names and class names.